features/vat-replacement/run.js

```diff
@@ -0,0 +1,235 @@
+    }
+    return map
+}
+
+async function getUpdateJsonObj(taxRateDraft, taxRateIdToTaxCategoryMap ) {
+    const taxCategoryItem = taxRateIdToTaxCategoryMap.get(taxRateDraft.id)
+
+    const updateJsonObj = {
+        version : taxCategoryItem.version,
+        actions : [
+            {
+                action: "replaceTaxRate",
+                taxRateId: taxRateDraft.id,
+                taxRate: {
+                    name: taxRateDraft.name,
+                    amount: taxRateDraft.amount,
+                    includedInPrice : taxRateDraft.includedInPrice,
+                    country: taxRateDraft.country
+                }
+            }
+        ]
+    }
+    return updateJsonObj
+
+}
+
+async function processTaxRate({ctpClient, taxRateDraftList, taxRateIdToTaxCategoryMap, oldTaxRate, newTaxRate,
+                                  isDryRun}) {
+    let clonedTaxRateDraftList = JSON.parse(JSON.stringify(taxRateDraftList))
+    for (const taxRateDraft of clonedTaxRateDraftList) {
+        taxRateDraft.amount  = newTaxRate
+        const updateJsonObj = await getUpdateJsonObj(taxRateDraft, taxRateIdToTaxCategoryMap)
+        if (!isDryRun) {
+            console.log('Start to replace tax rate ... ')
+            await replaceTaxRate(ctpClient, taxRateIdToTaxCategoryMap.get(taxRateDraft.id).id, updateJsonObj)
+            console.log('Update finished')
+        } else {
+            console.log('Current tax rate would be replaced as below : ')
+            console.log(JSON.stringify(updateJsonObj))
+        }
+    }
+}
+
+function printPreviewModeWarning() {
+    console.log('********************************************************************')
+    console.log('The script has been run in preview mode. To update the tax rates, ' + '\n' +
+        'you can input argument "-update" in CLI to run this script.')
+    console.log('Please make sure to resolve all tax categories and rates warnings ' + '\n' +
+        'before running the script in update mode.')
+    console.log('********************************************************************')
+}
+
+async function initConfigOptions() {
+    const args = process.argv.slice(2);
+    const configOptions = {
+        ctp : {
+            projectKey : process.env.PROJECT_KEY,
+            clientId : process.env.CLIENT_ID,
+            clientSecret : process.env.CLIENT_SECRET
+        },
+        dryRun: true
+    }
+
+    // Check environment variables
+    if (!configOptions.ctp.projectKey || !configOptions.ctp.clientId || !configOptions.ctp.clientSecret) {
+        console.error('Please set project key, client ID and client secret in environment variables')
+        return
+    }
+
+    // Check update mode / preview mode
+    if (args.length>0 && args[0] === '-update') {
+        configOptions.dryRun = false
+    }
+    return configOptions
+}
+
+(async function main() {
+    const configOptions = await initConfigOptions()
```

Comment:
You also don't need async/await here.

Reply:
https://github.com/commercetools/commercetools-cli-scripts/pull/1/commits/d39ef794b025359e7322ceffd6673a5c0d9303bd